stdlib/3/unittest.pyi

```diff
@@ -8,6 +8,7 @@ from typing import (
 import logging
 import sys
 from types import ModuleType, TracebackType
+from contextlib import ContextManager
```

Comment:
(Here I'm more OK -- we'll have to fix this once we add ContextManager to typing.py[i] for all PY versions, but that's a ways off.)